### PR TITLE
media-sound/bitwig-studio: bump to version 2.1.1

### LINKED ISSUE #8: bump bitwig studio to version 2

### PR DESCRIPTION
**Pull request by mrVanDalo**
Fixes #8

### CODE REVIEW

File: media-sound/bitwig-studio/bitwig-studio-2.1.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+	opt/bitwig-studio/lib/bitwig-studio/libXau.so.6
+	opt/bitwig-studio/lib/bitwig-studio/libXcursor.so.1
+	opt/bitwig-studio/lib/bitwig-studio/libXdmcp.so.6
+	opt/bitwig-studio/lib/bitwig-studio/libcairo.so.2
+	opt/bitwig-studio/lib/bitwig-studio/libfreetype.so.6
+	opt/bitwig-studio/lib/bitwig-studio/libharfbuzz.so.0
+	opt/bitwig-studio/lib/bitwig-studio/liblwjgl.so
+	opt/bitwig-studio/lib/bitwig-studio/libpng16.so.16
+	opt/bitwig-studio/lib/bitwig-studio/libxcb-xkb.so.1
+	opt/bitwig-studio/lib/jre/lib/amd64/libjfxwebkit.so"
```
Comment (simonvanderveldt) [inline media-sound/bitwig-studio/bitwig-studio-2.1.1.ebuild]: For some reason this one doesn't seem to be necessary anymore. I tried installing it first without anything in `QA_PRESTRIPPED` and then it threw warnings for all of these files except for `opt/bitwig-studio/lib/jre/lib/amd64/libjfxwebkit.so`
If you want you can remove it

Comment (mrVanDalo) [inline media-sound/bitwig-studio/bitwig-studio-2.1.1.ebuild]: sure

**Comment by mrVanDalo** [comment]
I fixed the QA_PRESTRIPPED list. there are still some files which need to be listed

**Comment by simonvanderveldt** [comment]
> there are still some files which need to be listed

@mrVanDalo You have QA warnings with the current ebuild?
I just tried it and I don't see any.

**Comment by mrVanDalo** [comment]
no I don't have any warnings anymore. (missunderstanding)

**Pull request merged**